compiler/rustc_passes/src/check_attr.rs

```diff
@@ -415,26 +413,21 @@ impl CheckAttrVisitor<'_> {
     }
 
     /// Debugging aid for `object_lifetime_default` query.
-    fn check_object_lifetime_default(&self, hir_id: HirId, span: Span) {
+    fn check_object_lifetime_default(&self, hir_id: HirId) {
         let tcx = self.tcx;
         if let Some(generics) = tcx.hir().get_generics(tcx.hir().local_def_id(hir_id)) {
```

Comment:
This could be replaced with `let`/`else { return; }`, but isn't necessary in this PR.